scripts/projectView.js

```diff
@@ -1,6 +1,58 @@
-// (function(module) {
+(function(module) {
   var projectView = {};
 
+  projectView.handleLanguageFilter = function() {
```

Comment:
The handleLanguageFilter and the handleLibraryFilter are to methods that can be refactored into one method that takes a few arguments.  Since each method uses in a drop down box HTML element and either refers to the languages or libraries array structure from the portfolioProjects object, both of these can be passed in as arguments for this one method.